R/commonMachineLearningClassification.R

```diff
@@ -645,9 +661,9 @@ gettextf <- function(fmt, ..., domain = NULL) {
     return()
   }
   if (nrow(dataset) > 500) {
-    sample <- sample(1:nrow(dataset), size = 500, replace = FALSE) # Sample to prevent crazy long loading times with big data
+    sample <- sample(seq_len(nrow(dataset)), size = 500, replace = FALSE) # Sample to prevent crazy long loading times with big data
```

Comment:
```suggestion
    sample <- sample(nrow(dataset), size = 500, replace = FALSE) # Sample to prevent crazy long loading times with big data
```
is there an issue that relates to this, or did you figure this out yourself? If it's the former, please leave the link as a comment.

Reply:
VS code linter gave this, but there is no issue for it. I guess it is more of a good practice

Reply:
Made this `sample <- sample.int(min(500, nrow(dataset)), size = 500, replace = FALSE)`

Reply:
What linter do you use? `seq_len` is better than `1:nrow(...)`, but `sample` also works if the first argument is an integer, in which case it does the same as `sample(seq_len(int)` but more efficient.

Reply:
Interesting, I actually didn't know that `sample` had this behaviour! I guess `sample.int` is even more efficient?

Let's see if the unit tests for the Andrews curves still pass after this small change.